test_files/test_bld.asm

```diff
@@ -0,0 +1,13 @@
+;-- Setting bits to true which are already true.
+ldi r31, 0xFF
+set
+bld r31, 6
+bld r31, 5
+bld r31, 4
+
+;-- Setting for no T flag.
+bclr 6
+ldi r30, 0x0F
+bld r30, 1
+bld r30, 3
+bld r30, 7
```

Comment:
These you can leave untouched, just make sure that it's clear that you would be doing the same thing as you did with r31, but you do it with r30 instead, and with 
T set to 0 this time. Obivously don't do both tests (T=1 and T=0) with the same register, as you would be overwriting the other test.